src/components/Trash/Trash.jsx

```diff
@@ -15,12 +17,11 @@ export const Trash = () => {
     dispatch({ type: "LOAD_NOTES", payload: res.data.notes });
   };
 
-  const addArchive = async (n, t) => {
-    const res = await addArchives({ note: n, noteId: n._id, encodedToken: t });
-    //console.log(res.data);
-    dispatch({ type: "DELETE_TRASH", payload: n._id });
-    dispatch({ type: "LOAD_ARCHIVES", payload: res.data.archives });
-  };
+  // const addArchive = async (n, t) => {
+  //   const res = await addArchives({ note: n, noteId: n._id, encodedToken: t });
+  //   dispatch({ type: "DELETE_TRASH", payload: n._id });
+  //   dispatch({ type: "LOAD_ARCHIVES", payload: res.data.archives });
+  // };
 
```

Comment:
remove these.

Reply:
I need to check that, then I will remove.